src/standalone/Snow/neural_snow/DataTools.jl

```diff
@@ -0,0 +1,549 @@
+module DataTools
```

Comment:
I am wondering if this particular module belongs in src vs in the docs/tutorials/Snow folder. The reason why is because this functionality is needed to create the dataset used in your paper, and has some handy function for processing data, but, nothing in here is required for evaluating a trained model to get the snow depth at the next step - which is what we need in the land model simulation (all that is in ModelTools.jl, which definitely belongs in src).

What do you think?

Reply:
I figured I was helping future work in case CliMA eventually wanted other types of SNOTEL data in training other models (albedo, runoff, etc., the module has all this functionality) and therefore wasn't specifically tied to this particular tutorial/use-case, and therefore should be in a location intended for wider usage. If you disagree I have no issues moving it to the tutorial folder since others can always go grab the tools from there anyway.